server/config/subject_page.proto

```diff
@@ -53,6 +53,8 @@ message EventTypeSpec {
   repeated DisplayProp display_prop = 6;
   // List of props that could be used to hold the end date for this event type
   repeated string end_date_prop = 7;
+  // The prop to use to get the geojson for each event
+  string geo_json_prop = 8;
```

Comment:
since we are separating polygons and paths - perhaps do that for the property too.

Reply:
good idea, done